cli/tests/078_unload_on_exit.ts

```diff
@@ -1,4 +1,9 @@
 window.onunload = () => {
   console.log("onunload is called");
+  // This second exit call doesn't trigger unload event,
+  // and therefore actually stops the process.
+  Deno.exit(0);
```

Comment:
It might also be good to use a different exit code here to help "prove" this worked?

Reply:
Sounds good! Updated.